grails-app/controllers/org/pih/warehouse/api/ProductClassificationApiController.groovy

```diff
@@ -0,0 +1,14 @@
+package org.pih.warehouse.api
+
+import grails.converters.JSON
+import org.pih.warehouse.product.ProductClassificationService
+
+class ProductClassificationApiController {
```

Comment:
There are multiple classification systems (taxonomies) related to products, so we should be thoughtful about what this might turn into and not pigeonhole product classification as ABC class. For example, product classifications represent categories, tags, ABC classes, UNSPSC, hazardous classes, narcotics classes. So I would either be very specific with the API or leave room for using this API for different classifications.

Reply:
yeah I'm not sure how to narrow this down because it feels like we're running out of words to describe the "type" of a product. Like you said, we have categories, tags, classes... and they're all used in different ways. My original idea for this was that it'd only be used for the "ABC class" style classifications (I didn't call it ProductABCClassApiController because we can support whatever prioritization ruleset that we want to use, not just ABC classification) but it does raise an interesting point that the line between all the different "types" is blurry and there might be some overlap.